filechecker.py

```diff
@@ -0,0 +1,17 @@
+import os
+import argparse
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('path', type=checkpath)
+    return parser.parse_args()
+
+def checkpath(path):
+    if os.path.exists(path):
```

Comment:
looking good, I would add a check if it is a directory (you do not want to read a directory)